### DISCUSSION TITLE
Incomming data and object counter (and size)

### DISCUSSION THREAD

**Discussion by BlackWand1** [minio/minio#18901] [Q&A]
Shouldn't the minio service keep track of incomming and deleted data live and be represented in the Object Browser?
Or is data just stored into the bucket and then the Scanner must do its job afterwards?

I even created a new bucket at least a week back, it is still not listed in the prometehus using "minio_bucket_usage_total_bytes" query.

**Comment by BlackWand1** [comment]
Just to clarify, the other (older) buckets are present in prometheus and the size value is increasing.

**Reply by klauspost** [reply]
> Shouldn't the minio service keep track of incomming and deleted data live 

No. Data usage is lazy and not a live representation.

**Reply by BlackWand1** [reply]
Wouldn't that put much trust in the scanner?
If we want correct numbers to be able to invoce our customers, if would be great if those numbers could be as accurate as possible.
And since I can other buckets increase "live", each time i press "refresh" in prometehus, shouldn't that "live" be for all buckets?

**Reply by klauspost** [reply]
@BlackWand1 Feel free to reach out to `sales@min.io` for your commercial concerns.
